### DISCUSSION TITLE
Cohesive elements with hexahedral elements

### DISCUSSION THREAD

**Discussion by JadeUoM** [neperfepx/neper#694] [General]
Hi everyone,

I am trying to create cohesive elements with hexahedral elements, but Neper cannot complete the process and displays the following messages:
   98%
   99%
  100%Info   :   - Processing mesh at interfaces...
  Info   :     > Making mesh discontinuous...

The code I used are:

neper -T -domain "cylinder(3, 5., 36)" -n "from_morpho" -morpho "gg(0.8)" -id 17 -ori "random" -oridescriptor euler-bunge -oriformat plain -regularization 1 -format "tess,ori" -o Neper_Test

neper -M Neper_Test.tess -elttype hex -rcl 0.4 -order 1 -meshqualmin 0.9 -interface cohesive -format "msh,inp" -mesh3dclreps 0.1

Thank you in advance.

Kind regards,
Jade

**Comment by rquey** [comment]
It's not implemented.  Neper2CAE seems to do it, see https://neper.info/others.html#others-unofficial.

**Reply by JadeUoM** [reply]
Many thanks, Romain.
